Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Destination/UpdateDestination.cs

```diff
@@ -71,6 +82,16 @@ public static class UpdateDestination
 
             return result;
         }
+        
+        private static bool removeDestination(string type, dynamic data)
+        {
+            bool answer = (type.Equals(Constants.NOTIFICATION_TYPE_LOCATION) && 
+                           (data.location.longitude == null || data.location.latitude == null)) ||
+                           (type.Equals(Constants.NOTIFICATION_TYPE_WIFI) && data.location.ssid == null) ||
+                           (type.Equals(Constants.NOTIFICATION_TYPE_BLUETOOTH) && data.location.device == null);
```

Comment:
1. Split this bool into several bools. It's not very readable this way..
2. Please rename "answer" with "result" or "canBeRemoved" or something like that, indicating what this method does.
3. In general, if the method's name is "removeDestination", and it doesn't remove anything, but just checking something - use another name.